src/Makefile

```diff
@@ -323,13 +323,27 @@ clean-support:
 
 cleanall: clean clean-flisp clean-support
 
-analyzegc:
+$(build_shlibdir)/libGCCheckerPlugin.$(SHLIB_EXT): $(SRCDIR)/clangsa/GCChecker.cpp
+	@$(call PRINT_CC, $(CXX) -g -shared -fno-rtti -fPIC -o $@ -std=c++11 -DCLANG_PLUGIN -I$(build_includedir)  -I$(LLVM_SRC_DIR)/include -L$(build_libdir) $< -lclangAnalysis -lclangStaticAnalyzerCore -lclangASTMatchers -lclangAST -lclangLex -lclangBasic)
```

Comment:
seems like most of these flags are supposed to be coming from `llvm-config`, but got entered manually here

Reply:
@Keno can you confirm or deny?

Reply:
That'd probably work.

Reply:
I took at shot at this; let me know if I did something wrong; we're passing an awful lot more flags through now, but it seems to work from my perspective.